pkg/cmd/attestation/verification/sigstore.go

```diff
@@ -93,6 +152,18 @@ func (v *LiveSigstoreVerifier) chooseVerifier(b *bundle.ProtobufBundle) (*verify
 	return nil, "", fmt.Errorf("leaf certificate issuer is not recognized")
 }
 
+func getLowestCertInChain(ca *root.CertificateAuthority) (*x509.Certificate, error) {
+	if ca.Leaf != nil {
+		return ca.Leaf, nil
+	} else if len(ca.Intermediates) > 0 {
+		return ca.Intermediates[len(ca.Intermediates)-1], nil
```

Comment:
Here I think the the lowest intermediate would be `ca.Intermediates[0]`. The chain is order from so that the leaf is first, then any intermediates (where the successor signs the previous) and then the root.

Reply:
```suggestion
		return ca.Intermediates[0], nil
```